### PR TITLE
Prep 3.0.0-beta.0 release

### PR DESCRIPTION
**Pull request by giladgray**
(replacing #2473 so we can use Lerna on `next` branch for this release.)

### CODE REVIEW

**Comment by llorca** [comment]
should we audit hardcoded CSS markups? I think this one is inaccurate, uses `h5` without a class, right? https://14798-71939872-gh.circle-artifacts.com/0/home/circleci/project/packages/docs-app/dist/index.html#core/components/card.javascript-api

**Comment by JKillian** [comment]
👍 code lgtm!

File: packages/core/src/components/html/html.md [hunk 1]
```diff
@@ -7,11 +7,11 @@ reference: html
...
 most HTML elements directly. Instead, we provide several ways to style basic elements:
 
-1. Use Blueprint React components: `<H1>`
-1. Apply Blueprint the `Classes` constant to an HTML tag: `<h1 className={Classes.HEADING}>`
+1. Use Blueprint React components: `<H1>`.
+1. Apply the Blueprint `Classes` constant to an HTML tag: `<h1 className={Classes.HEADING}>`.
 1. Nest HTML tags inside a container with `Classes.RUNNING_TEXT` (see below).
 
-The following elements should be used in this manner:
+The following elements should be used in the following way:
```
Comment (giladgray) [inline packages/core/src/components/html/html.md]: but it's not the `following` manner -- it's described above

File: packages/docs-app/src/styles/_sections.scss [hunk 2]
```diff
@@ -262,6 +262,22 @@
   }
 }
 
+// give all Table examples a fixed height
+#{page("html")} {
```
[the remark this answers is not in the document]
Comment (giladgray) [reply packages/docs-app/src/styles/_sections.scss]: also the comment is incorrect

[the remark this answers is not in the document]
Comment (giladgray) [reply packages/docs-app/src/styles/_sections.scss]: @llorca fixed in new commit

**Pull request merged**